engine/engine.go

```diff
@@ -268,6 +269,12 @@ func (e *Engine) runTask(ctx context.Context, task tasks.Task, runCount, worker
 		}
 		tlog.Errorw("error updating final status", "err", err)
 	}
+	if e.nodeConfig.PostHook != "" {
+		err := exec.Command("/bin/bash", e.nodeConfig.PostHook, task.GetUUID()).Run()
```

Comment:
you mean PostHook to be a script file, not a command, right? if so, I'd document the flag as a script, not a command. otherwise I'd expect to give it `--posthook="some command"`, not `--posthook=/path/to/some-command.sh`.